tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature

```diff
@@ -162,22 +162,6 @@ Feature: Sharing files and folders with internal groups
     Then file "lorem.txt" should be listed on the webUI
     And as "Carol" the content of "new-simple-folder/lorem.txt" should be the same as the original "simple-folder/lorem.txt"
 
-  @issue-5015
```

Comment:
why deleting these tests?

Reply:
should cover by https://github.com/owncloud/web/blob/e1b958326be41975cd53445a6d88bcf92f3be1e2/tests/acceptance/features/webUISharingInternalGroups/shareWithGroups.feature#L200-L209